arch/configure.defaults

```diff
@@ -304,12 +304,12 @@ DESCRIPTION     =       INTEL ($SFC/$SCC)
 DMPARALLEL      =       # 1
 OMPCPP          =       # -D_OPENMP
 OMP             =       # -qopenmp -fpp -auto
-OMPCC           =       # -qopenmp -fpp -auto
-SFC             =       ifort
-SCC             =       icc
-CCOMP           =       icc
-DM_FC           =       mpif90 -f90=$(SFC)
-DM_CC           =       mpicc -cc=$(SCC)
+OMPCC           =       # -qopenmp
+SFC             =       ifx # ifort
+SCC             =       icx # icc
+CCOMP           =       icx # icc
+DM_FC           =       mpiifx #mpif90 -f90=$(SFC)
+DM_CC           =       mpiicx #mpicc -cc=$(SCC)
```

Comment:
@ahheo Should not change this option. Add another option like it is in PR#[1893](https://github.com/wrf-model/WRF/pull/1893).

Reply:
OK. Well, what should I do with this PR for the next step? Should I convert it as a draft and make a new commit and submit this PR again?

Reply:
@ahheo Please continue to modify this PR.

Reply:
@weiwangncar I have made this modification following your suggestion: restoring the changed option and adding another option as PR #1893 , by another commit [7361882](https://github.com/wrf-model/WRF/pull/1942/commits/736188253339eacaf664a6d1e90f3cb8e679b115). Well, the commit still failed in WRF regression test, and I was not able to check the details. Sorry that this is my first time making a PR, and I am still learning the procedures to make things work.

Reply:
@ahheo The failure of the regression tests is not your fault. We currently have some trouble running the tests on the Cloud. We are working to resolve the issues.